tests/Database/DatabaseFunctionalTest.php

```diff
@@ -79,7 +79,7 @@ public function testCommand(): void
 
         $cursor = $this->database->command($command, $options);
 
-        $this->assertInstanceOf(Cursor::class, $cursor);
+        $this->assertInstanceOf(CursorInterface::class, $cursor);
```

Comment:
Noted we're not bothering with additional checks for `Iterator::class` since it'd just become irrelevant after CursorInterface extends it in 2.0.